src/pudl/output/ferc1.py

```diff
@@ -2496,11 +2492,16 @@ def forest_leaves(self: Self) -> list[NodeId]:
     def orphans(self: Self) -> list[NodeId]:
         """Identify all nodes that appear in metadata but not in the full digraph."""
         nodes = self.full_digraph.nodes
-        return [
-            NodeId(*n)
-            for n in self.exploded_meta.set_index(self.calc_cols).index
-            if n not in nodes
-        ]
+        orphans = []
+        for idx_cols in [self.calc_cols, self.parent_cols]:
+            orphans.extend(
+                [
+                    NodeId(*n)
+                    for n in self.exploded_calcs.set_index(idx_cols).index
+                    if n not in nodes
```

Comment:
oh this is a good question... This has definitely picked up orphans without the `NodeId(*n) not in nodes` so i assume it works as is but i don't think it would hurt to add this guy (on a broader note i am already thinking about ways to adopt the unit test setup we added here to test other things o/s of tag propagation like this or #3309

Reply:
nit: did you still want to add this? either way is fine w/ me

Reply:
i didn't transition it bc it seemed hard to test initially. but i rejiggered the tests a bit to make sure i could triple check that this was going to work okay. turns out the change in using the `exploded_calcs` here instead of the old metadata table makes this whole `orphans` thing I believe redundant and unnecessary. but i'd like to double check that with zane before ripping it out.